Linux SSL instructions and other doc tweaks.

blackbaud-johnly opened the pull request.

blackbaud-johnly commented:
Addresses https://github.com/blackbaud/skyux2/issues/1281. I sent Matt a screenshot of the new SSL instructions to get his feedback.

Blackbaud-TerryHelems commented on src/app/learn/get-started/prereqs/ssl-certificate/index.html:
```diff
@@ -12,11 +12,11 @@
-        <li><p>Expand the Trust section near the top of the info window, and select the "Always Trust" option for "Secure Sockets Layer (SSL)."</p></li>
+        <li><p>Double-click the new <strong>SKY UX</strong> item in the list to open the certificate's info window.</p></li>
+        <li><p>Expand the Trust section near the top of the info window, and select the <strong>Always Trust</strong> option for Secure Sockets Layer (SSL).</p></li>
         <li><p>Close the info window.</p></li>
```
We don't want `SSL` emphasized?

blackbaud-johnly replied:
I'll emphasize it.

Blackbaud-TerryHelems commented on src/app/learn/get-started/prereqs/ssl-certificate/index.html:
```diff
@@ -25,47 +25,57 @@
+    <ol>
+        <li><p>Download the <a href="https://raw.githubusercontent.com/blackbaud/skyux-builder/master/ssl/skyux-ca.crt">skyux-ca.crt</a>.</p></li>
+        <li><p>Move the <stache-code>skyux-ca.crt</stache-code> to <stache-code>/usr/local/share/ca-certificates</stache-code>.</p></li>
+        <li><p>From the command prompt, run <stache-code>sudo update-ca-certificates</stache-code>.</p>
```
Does "command prompt" sound like Windows?  I would have though "terminal" for a Linux person, but maybe not?

blackbaud-johnly replied:
I'll change it to "terminal."

Blackbaud-BobbyEarl commented on src/app/learn/get-started/prereqs/ssl-certificate/index.html:
```diff
@@ -25,47 +25,57 @@
+        <li><p>Download the <a href="https://raw.githubusercontent.com/blackbaud/skyux-builder/master/ssl/skyux-ca.crt">skyux-ca.crt</a>.</p></li>
+        <li><p>Move the <stache-code>skyux-ca.crt</stache-code> to <stache-code>/usr/local/share/ca-certificates</stache-code>.</p></li>
+        <li><p>From the command prompt, run <stache-code>sudo update-ca-certificates</stache-code>.</p>
+          <sky-alert alertType="info">When you use products that use their own certificate stores, you also need to add the certificate to those certificate stores. For example, follow the instructions above to install the certificate for Firefox, <a href="http://blogmines.com/blog/how-to-import-a-certificate-in-google-chrome/">import the certificate in Google Chrome</a>, or <a href="https://docs ...
```
Typo in the very last word.  "tore"

blackbaud-johnly commented:
I addressed the 2 items @Blackbaud-TerryHelems pointed out and fixed the "tore" typo that @Blackbaud-BobbyEarl caught. Thanks for the feedback. ... I'll hold off on merging for just a little while to see if Matt has any additional feedback.
